core/unit_test/TestTeamVector.hpp

```diff
@@ -869,7 +869,7 @@ TEST(TEST_CATEGORY, team_vector) {
 #endif
 
 // FIXME_HIP
-#if !defined(KOKKOS_IMPL_CUDA_CLANG_WORKAROUND) && !defined(KOKKOS_ENABLE_HIP)
+#if !defined(KOKKOS_IMPL_CUDA_CLANG_WORKAROUND)
```

Comment:
Remove the `FIXME_HIP`

Reply:
774d6e75135122cab5f597855f7bd556fb77f36c